### PR TITLE
Fix make for CI

### PR DESCRIPTION
**Pull request by clarketm**
see: https://github.com/istio/common-files/pull/207#discussion_r390026663 for d415621.
see: https://github.com/istio/common-files/pull/208 for 4533a0d

### CODE REVIEW

**Comment by howardjohn** [comment]
feel free to remove the hold

File: files/Makefile [hunk 1]
```diff
@@ -54,9 +54,9 @@ else
 # If we are not in build container, we need a workaround to get environment properly set
 # Write to file, then include
 $(shell mkdir -p out)
-$(shell $(PWD)/common/scripts/setup_env.sh envfile > out/.env)
+$(shell $(shell pwd)/common/scripts/setup_env.sh envfile > out/.env)
 include out/.env
-export out/.env
+export
```
Comment (sdake) [inline files/Makefile]: is this stray debug?

Comment (clarketm) [reply files/Makefile]: It is to export the included variables from the previous line. I think this was the original intention of `export out/.env`.

Comment (clarketm) [reply files/Makefile]: Thanks for documenting this behavior in: https://github.com/istio/common-files/pull/211

**Pull request merged**